src/layout/RootLayout.tsx

```diff
@@ -28,6 +27,7 @@ export function RootLayout(props: RootLayoutProps) {
   }, []);
 
   return (
+    // eslint-disable-next-line no-warning-comments
```

Comment:
```suggestion
```

Warnings for TODO comments should never be disabled. They are here so we don't forget the TODO.

Reply:
resolved